plugins/modules/k8s_json_patch.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: k8s_json_patch
+
+short_description: Apply JSON patch operations to existing objects
+
+description:
+  - This module is used to apply RFC 6902 JSON patch operations only.
+  - Use the C(k8s) module for strategic merge or JSON merge operations.
+  - The jsonpatch library is required for check mode.
+
+version_added: 2.0.0
+
+author: Mike Graves (@gravesm)
+
+options:
+  api_version:
+    description:
+      - Use to specify the API version.
+      - Use in conjunction with I(kind), I(name), and I(namespace) to identify a specific object.
+    type: str
+    default: v1
+    aliases:
+      - api
+      - version
+  kind:
+    description:
+      - Use to specify an object model.
+      - Use in conjunction with I(api_version), I(name), and I(namespace) to identify a specific object.
+    type: str
+    required: yes
+  namespace:
+    description:
+      - Use to specify an object namespace.
+      - Use in conjunction with I(api_version), I(kind), and I(name) to identify a specific object.
+    type: str
+  name:
+    description:
+      - Use to specify an object name.
+      - Use in conjunction with I(api_version), I(kind) and I(namespace) to identify a specific object.
+    type: str
+    required: yes
+  patch:
+    description:
+      - List of JSON patch operations.
+    required: yes
+    type: list
+    elements: dict
+
+extends_documentation_fragment:
+  - kubernetes.core.k8s_auth_options
+  - kubernetes.core.k8s_wait_options
+
+requirements:
+  - "python >= 3.6"
+  - "kubernetes >= 12.0.0"
+  - "PyYAML >= 3.11"
+  - "jsonpatch"
+'''
+
+EXAMPLES = r'''
+- name: Apply multiple patch operations to an existing Pod
+  kubernetes.core.k8s_json_patch:
+    kind: Pod
+    namespace: testing
+    name: mypod
+    patch:
+      - op: add
+        path: /metadata/labels/app
+        value: myapp
+      - op: replace
+        patch: /spec/containers/0/image
+        value: nginx
+'''
+
+RETURN = r'''
+result:
```

Comment:
`result` is a reserved keyword. I would suggest changing it to something meaningful like `patched_output` or `json_output`.